### ISSUE TITLE
com.microsoft.azure:azure-mobile-android:3.4.0 should not set android:allowBackup to true 

### ISSUE THREAD

**Issue by MMLino** [Azure/azure-mobile-apps-android-client#155]
The library **cordova-plugin-ms-azure-mobile-apps** is setting android:allowbackup=true and this is not a correct behavior for a library manifest. I need to have, for a release cordova build, android:allowBackup=false and for this reason I also tried to set it via edit-config command also adding the attribute tools:replace="android:allowBackup" and tools:node="replace".
Below an example:

```
<platform name="android">
	[...]
	<edit-config file="app/build/intermediates/manifests/full/release/AndroidManifest.xml" mode="merge" target="/manifest/application">
		<activity android:allowBackup="false" tools:node="replace" tools:replace="android:allowBackup" />
	</edit-config>
</platform>
```
The result is:
FAILURE: Build failed with an exception.

```
* What went wrong:
Execution failed for task ':app:processReleaseManifest'.
> Manifest merger failed : Attribute application@allowBackup value=(false) from AndroidManifest.xml:5:18-45
        is also present at [com.microsoft.azure:azure-mobile-android:3.4.0] AndroidManifest.xml:11:18-44 value=(true).
        Suggestion: add 'tools:replace="android:allowBackup"' to <application> element at AndroidManifest.xml:5:5-25:19 to override.
```

**Comment by AnastasiaKubova** [comment]
Thanks for getting in touch with us! 
It looks like a bug. I mark this issue as a bug on our board.

**Comment by adrianhall** [comment]
Updated the AndroidManifest to allow replacement, implemented in PR #156.  Will have to wait on release to bintray for the Cordova client update.

**Comment by MMLino** [comment]
HI all, sorry for the delay in my response. 
Thank you all for the clarification.
